web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 # Aggregation
 
 A key feature of analytics is reducing many values down to some summary. This
-act is called aggregation and always includes an aggregation function, for
-example `average` or `sum`:
+act is called "aggregation" and always includes a function &mdash; for
```

Comment:
Can we just use a dash in markdown?

```suggestion
act is called "aggregation" and always includes a function — for
```

Reply:
I don't think Markdown will emit an em dash (which is typographically more correct) for a bare "-" in the source. I'm OK either way - we can add the decision to our style guide.

Reply:
FYI on Mac this is just `alt+shift+-`: —